docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,176 @@
+<<mounting-properties-files,mount the properties files>> to the Kubernetes resource allowing the configuration store to access the custom properties. Additionally,
+the <<backend-storage-types,backend storage type>> can be adjusted to fit a project's needs.
+
+==== Mounting Properties Files
+Properties files are attached to the local file system of the configuration store within Kubernetes through a mounted volume. To mount
+a volume to the configuration store, one must set the helm chart's https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md#persistent-volume-properties[persistent volume properties,role=external,window=_blank]. Then the helm chart's https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md#environment-variables[environment 
+variables,role=external,window=_blank] must be updated with the corresponding URI(s) to the properties files within that volume. It is required to specify 
+a base property URI that houses the base/default property configurations. In addition to the base URI, one can optionally specify a 
+secondary URI for properties files that house the environment specific configurations. 
+
+NOTE: Multiple configuration properties files can be stored at either URI.
+
+=== Backend Storage Types
+By default, the configuration store uses Krausening for it backend property storage. This provides a more light weight but still robust 
+storage option perfect for smaller applications.
+
+For enterprise applications that have more robust requirements, it is recommended to utilize https://developer.hashicorp.com/vault[Vault,role=external,window=_blank]. Additionally, the configuration 
+store can be set up to use in memory storage, this is best for quick local testing.
+
+To change it to use one of the other options, set the environment variable `STORAGE_CLASS` to the desired value. The values currently
+available are `krausening`, `vault`, and `inMemory`.
+
+== Usage
+After the configurations are ingested and reconciled within the store, the next step is to access them within a project using one of 
+the two available methods: the <<rest-endpoint,REST endpoint>> or <<kubernetes-resource-injection-recommended,Kubernetes resource injection>>.
+Configuration properties are identified by using a group name and property name, where group name is determined by the name of the file
+the property is defined in.
+
+Given an example properties file named `messaging.properties`:
+[source,properties]
+----
+connector=smallrye-kafka
+----
+
+The property name would be `connector` and group name would be `messaging`.
+
+=== REST Endpoint
```

Comment:
Q: Do we have a client API that we want to reference here? (Can't recall if we created one yet.)

Reply:
At this time we do not